src/line.fragment.glsl

```diff
@@ -1,25 +1,28 @@
-uniform lowp float u_opacity;
-uniform float u_blur;
-
 #pragma mapbox: define lowp vec4 color
+#pragma mapbox: define lowp float blur
+#pragma mapbox: define lowp float opacity
+#pragma mapbox: define mediump float gapwidth
```

Comment:
`gapwidth` doesn't appear to be used in the fragment shader -- I think we can remove.